linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_init/ansible/shared.yml

```diff
@@ -4,12 +4,18 @@
 # disruption = low
 # strategy = configure
 
-{{% if "ol" in product or 'rhel' in product %}}
+{{% if product in ['ol', 'rhel', 'ubuntu2004'] %}}
```

Comment:
you are missing again to fix the bash remediation. 
if you change a remediation, then you need to fix all the different supported scripts. 
otherwise the rule will fail as it is right now in this branch

Reply:
also 'ol' and 'rhel' are not product names but part of the product names, so your change will break things for those vendors.
also for use 'ubuntu' in product and review all the ones you changed